oap-server/analyzer/log-analyzer/src/main/java/org/apache/skywalking/oap/log/analyzer/dsl/spec/filter/FilterSpec.java

```diff
@@ -161,9 +169,19 @@ public void sink(@DelegatesTo(SinkSpec.class) final Closure<?> cl) {
             return;
         }
 
+        if (extraLog != null) {
+            logData.setBody(
+                LogDataBody.newBuilder()
+                           .setJson(
+                               JSONLog.newBuilder()
+                                      .setJson(toJSON(extraLog))
+                                      .build())
+                           .build()
+            );
+        }
```

Comment:
Is this the last step before persistent?

Reply:
> Is this the last step before persistent?

Yes, it's last step of the LAL, maybe I can move it to right before persistence.

Reply:
Could use have multiple LAL files? If so, let's move it out of LAL scope.

Reply:
DONE